app/components/dev_card_component.html.erb

```diff
@@ -0,0 +1,23 @@
+<li>
```

Comment:
Let's move this tag to the index. The component might not always be rendered as a list item.